tests/Hangfire.Tags.PostgreSql.Tests/IntegrationTestBase.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Hangfire.Tags.PostgreSql.Tests
+{
+    public class IntegrationTestBase
+    {
+        protected IConfiguration Configuration { get; }
+        protected IServiceCollection Services { get; }
+
+        protected IntegrationTestBase()
+        {
+            Configuration = new ConfigurationBuilder()
+                .AddJsonFile("testsettings.json", optional: true)
+                .Build();
+
+            Services = new ServiceCollection();
+        }
+    }
```

Comment:
This is a big question because I don't really know how to make it work on your specific build environment since it is an integration test. If you can spin up a Postgres instance and have it cleaned up for each integration test run, it would be great.

Otherwise, I suggest that we keep that test, but disable it on the CI environment by putting integration tests under `IntegrationTests` category and filtering it out from `appveyor` script.

Reply:
Apparently, `appveyor` supports [PostgreSQL](https://www.appveyor.com/docs/services-databases/#postgresql) on every build machine. Let's just try it ;)